interchain-token-service/README.md

```diff
@@ -0,0 +1,15 @@
+# Interchain Token Service Hub
```

Comment:
would be nice to have a simple diagram in here, but can do at a later time

Reply:
yeah, planning to add more docs (especially with the balance tracking feature), until then the notion doc is the reference